### DISCUSSION TITLE
PayLog

### DISCUSSION THREAD

**Discussion by charvyCZ** [Zrips/CMI#3377] [Feature Requests]
Hello, would it be possible for you to either add paylog or PlayerPayEvent to API? We would like to track the sender, receiver, time and value of money sent using /pay command.

**Comment by Zrips** [comment]
Have you looked into CMIUserBalanceChangeEvent?

**Comment by charvyCZ** [comment]
Thanks for swift response. Yeah, I checked it out, but it isn't really usable as pay log, since it throws events for withdrawal and deposit separately. I'd like to have single event, that would include both sender and receiver along with other, already mentioned, properties.
